fix_includes_test.py

```diff
@@ -41,6 +41,7 @@ def __init__(self):
     self.invoking_command_line = 'iwyu.py my_targets'
     self.find_affected_targets = True
     self.keep_iwyu_namespace_format = False
+    self.reorder = True
```

Comment:
It's a little annoying that the default is True in tests, but False in production... But we don't want to update all tests as part of this change set anyway, so let's keep this as-is for now